src/components/ListItem/RadioButtonListItem/RadioButtonListItem.css

```diff
@@ -0,0 +1,99 @@
+@import "../../../styles/theme/index.css";
+
+:root {
+  --list-item-rb-flex-direction: row-reverse;
+
+  --list-item-rb-width--large: var(--heading-s);
+  --list-item-rb-height--large: var(--heading-s);
+
+  --list-item-rb-width--medium: var(--heading-xs);
+  --list-item-rb-height--medium: var(--heading-xs);
+
+  --list-item-rb-span-background: var(--structure-color-white);
+  --list-item-rb-span-background-color: var(--structure-color-300);
+  --list-item-rb-span-background-color-hover: var(--structure-color-white);
+  --list-item-rb-span-background-color-checked: var(--primary-color-400);
+  --list-item-rb-span-border: 1px solid var(--success-color);
+  --list-item-rb-span-border-checked: none;
+}
+
+.radio-div {
```

Comment:
Please rename this to `radio-container`. This would help in keeping a pattern across the lib, where we avoid the name of HTML tags in css